manifests/service.pp

```diff
@@ -5,10 +5,53 @@
 #
 class reaktor::service {
 
+  $user  = $::reaktor::user
+  $group = $::reaktor::group
+  $dir   = $::reaktor::_dir
+
+  case $::reaktor::service_provider {
+    'upstart': {
+      $init_mode = '0644'
+      $init_file_reaktor = "${::reaktor::init_dir}/reaktor.conf"
+      $init_file_reaktor_redis   = "${::reaktor::init_dir}/reaktor-redis.conf"
+
+      file { [
+        '/etc/init.d/reaktor',
+        '/etc/init.d/reaktor-redis'
+        ]:
+        ensure => 'link',
+        target => '/lib/init/upstart-job',
+      }
+    }
+    default: {
+      fail('Please, specify one of the following implementd service providers: upstart')
+    }
+  }
+
+  $default_config = {
+    'RACK_ROOT'                   => $dir,
+    'RESQUE_WORKER_USER'          => $user,
+    'RESQUE_WORKER_GROUP'         => $group,
+    'REAKTOR_PUPPET_MASTERS_FILE' => "${dir}/masters.txt",
+  }
+  $config = merge($default_config, $::reaktor::config)
+
+  file { $init_file_reaktor_redis:
+    ensure  => $ensure,
```

Comment:
hm, i'm a blind? i can't seem to find the definition of $ensure in class reaktor::service.